Controller/ContentManagement/PublishController.php

```diff
@@ -102,42 +105,36 @@ public function publishSearchResult(Request $request)
         $body = $this->getSearchService()->generateSearchBody($search);
         $form = $builder->getForm();
         $form->handleRequest($request);
-        
+
+        $body['query']['bool']['must'] = array_merge($body['query']['bool']['must'] ?? [], [['term' => [EMSSource::FIELD_CONTENT_TYPE => $contentType->getName()]]]);
```

Comment:
It might break old revisions records without the _contenttype field provided . We need a ems:revision:upgrade command to ensure that all elasticms internal fields are setup as expected (signed, _contenttype, ...)